Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.9.2-slim-buster as base
+RUN pip install poetry
+COPY poetry.lock pyproject.toml ./
+
+FROM base as production
+RUN poetry add gunicorn
+RUN poetry install
```

Comment:
For production you can do `poetry install --no-dev` to skip things that the image won't need, like pytest/selenium.

If you move those to the `tool.poetry.dev-dependencies` section of pyproject.toml